web_scraping.py

```
@@ -0,0 +1,92 @@
+                  "AppleWebKit/537.36 (KHTML, like Gecko) "
+                  "Chrome/140.0.0.0 Safari/537.36"
+}
+
+def fetch_page(url):
+    try:
+        print("Fetching page...")
+        session = requests.Session()
+        response = session.get(url, headers=HEADERS, timeout=10)
+        response.raise_for_status()
+        print("Page fetched successfully!")
+        return response.text
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)